Legion.FrontEnd/src/events/EventsCalendar/Components/DaysOfWeek.jsx

```diff
@@ -0,0 +1,20 @@
+/* Renders the column titles as the days of the week Sunday to Saturday */
+import React from 'react';
+import moment from 'moment';
+
+class DaysOfWeek extends React.PureComponent {
+  listDays = () => {
+    var names = []
+    for (var i = 0; i < 7; i++) {
+      names.push(<span className='column column-center' key={i}>{moment().day(i).format('ddd')}</span>)
```

Comment:
Don't use indexes for React keys - try to find something more unique.  In this case, you could use `dayName-month#-year#`.  For more reading: https://reactjs.org/docs/lists-and-keys.html#keys

Reply:
Yeah. I was having some problems with unique keys being repeated. Some warnings that I did not know how to deal with. I'll try this.